iothub_client/tests/iothubclient_uploadtoblob_e2e/iothubclient_uploadtoblob_e2e.c

```diff
@@ -340,6 +358,7 @@ void e2e_uploadtoblob_multiblock_test(IOTHUB_CLIENT_TRANSPORT_PROVIDER protocol,
 
 void e2e_uploadtoblob_test_multiple_simultaneous_uploads(IOTHUB_CLIENT_TRANSPORT_PROVIDER protocol)
 {
+    static int fileCounter = 0;
```

Comment:
>fileCounter [](start = 15, length = 11)

this doesn't seem to be doing anything. am i missing something here?

Reply:
It is static, so it will get 0 only on the first call.

---
In reply to: [324815469](https://github.com/Azure/azure-iot-sdk-c/pull/1174#discussion_r324815469) [](ancestors = 324815469)